deps/install_tensorflow_1_15.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
```

Comment:
See the comment above. I think we don't need the file.

Reply:
Ok